.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: go
 
 go:
-  - 1.8.x
-  - 1.9.x
+  - "1.11.x"
+  - master
 
 before_install:
+  - export GO111MODULE=on
```

Comment:
That's messing with the local environment and shouldn't be done.
This will need to be done manually or the folder you work in can be outside of GOPATH with go11.